net/c-icap-server/Makefile

```diff
@@ -0,0 +1,113 @@
+#
+# Copyright (C) 2006-2014 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=c-icap-server
+PKG_SOURCE_NAME:=C_ICAP
```

Comment:
There's no need for this

Reply:
It seemed complicated to me.

The download is available as C_ICAP_0.5.7.tar.gz. On unarchiving the directory I get is c-icap-server-C_ICAP_0.5.7/ which translated to:

PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_SOURCE_NAME)_$(PKG_VERSION)

else, I was not able to build the package.

Also, I wanted to avoid keeping package name in ALL CAPS.

If you have suggestion to handle this situation, please help.